### DISCUSSION TITLE
Enforce https

### DISCUSSION THREAD

**Discussion by magicmatt007** [homeassistant-apps/app-cloudflared#197] [Ideas]
First of all thank you for this great addon! This finally allowed me to get remote access, as my duckdns & port fowarding setup didn't work anymore with my new ISP.

Just one thing: I realized, that I can access my HA also via http. But then I found a setting on Cloudfare, which automatically switches those requests to https. See screenshot

Maybe it would make sense to have this automatically switched on via your plugin (if possible)....

Thx!

![2022-09-29_18h35_09](https://user-images.githubusercontent.com/58187053/193091415-fdf26820-c63d-4bc4-80d9-557fa7848ed5.png)

**Comment by skynet01** [comment]
I don't think its feasable for 2 things. The plugin is just a wrapper around the official Cloudflared addon so they control the functionality, also I think ( I might be wrong though) it only is allowed to setup tunnels and not configure your cloud flare account ... I do agree with you that there should be some sort of a basic guide / faq on this git that covers things like this
